request/request.go

```diff
@@ -84,6 +87,10 @@ func Request(method, url string, body io.Reader, headers map[string]string) (*ht
 		}
 	}
 
+	if userAgent != "" {
+		req.Header.Set("User-Agent", userAgent)
+	}
+	
```

Comment:
request/request.go:93: File is not `goimports`-ed (goimports)

You need to fix the lint warning.